lib/runner/run.js

```diff
@@ -58,6 +58,7 @@ module.exports = new (function() {
         errors  : results.errors,
         skipped : results.skipped,
         assertions  : [].concat(results.tests),
+        errorsMessages: (Array.isArray(errors) && errors.length) ? errors.slice() : null,
```

Comment:
`errors` is always an array so this can just be: 'errorsMessages: errors.slice()`.